pages/team.html

```diff
@@ -0,0 +1,143 @@
+<script>
+	{
+		"title": "The jQuery Team",
+		"customFields": [
+			{
+				"key": "subtitle",
+				"value": "The jQuery Project is run by a distributed group of volunteers."
+			},{
+				"key": "full-width",
+				"value": 1
+			}
+		]
+	}
+</script>
+
+<h1 class="block">The jQuery Team</h1>
+
+<ul class="block-grid five-up mobile-two-up team-list">
+	<li>
+		<img src="https://secure.gravatar.com/avatar/f16d4602c1c90646438a0b534ff61889?size=120" alt="">
+		<h2>Richard Gibson</h2>
+		<h3>Sizzle Lead | Core | QUnit</h3>
```

Comment:
I wonder if we should include Sizzle considering we're archiving it soon.

Reply:
Do you have a preferred way to formulate past roles of current members?

Reply:
I'd write:
```suggestion
		<h3>Core | QUnit | Former Sizzle Lead</h3>
```
but I'd also like to hear what @gibson042 thinks since it's under his name.

Reply:
Works for me.